clang/include/clang/Sema/Sema.h

```diff
@@ -4453,9 +4453,9 @@ class Sema final : public SemaBase {
                                       SourceLocation *ArgLocation = nullptr);
 
   /// Determine if type T is a valid subject for a nonnull and similar
-  /// attributes. By default, we look through references (the behavior used by
-  /// nonnull), but if the second parameter is true, then we treat a reference
-  /// type as valid.
+  /// attributes. By default, we skip dependence and look through references
```

Comment:
`by default` implies that the dependence check can bie disabled, but it very much cannot.

Reply:
Thank you, all comments addressed